barretenberg/cpp/src/barretenberg/dsl/acir_format/recursion_constraint.cpp

```diff
@@ -358,6 +354,13 @@ std::vector<barretenberg::fr> export_dummy_transcript_in_recursion_format(const
     return fields;
 }
 
+size_t transcript_in_recursion_format_size()
```

Comment:
I think we should rename this to be something like `recursion_proof_size_without_public_inputs`